### DISCUSSION TITLE
Keep Trivy Updated With Python

### DISCUSSION THREAD

**Discussion by Marcusdclark12** [aquasecurity/trivy#4732] [Q&A]
### Question

Hello! I am currently working with Trivy and want to know is there a way to have trivy check if it is the latest version in python without me having to hard code the latest version? Like how would a function be made in python inorder to keep trivy up to date?
Thank you!

### Target

AWS

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Operating System

Linux

### Version

```bash
Version: 0.42.1
Policy Bundle:
  Digest: sha256:9db84b217f767f81ec98ca40c94e81562dbd6cf5f10083da236322ed99eecfda
  DownloadedAt: 2023-06-28 19:55:25.10652117 +0000 UTC
```

**Comment by knqyf263** [comment]
Do you need this API?
https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release
